zstacklib/zstacklib/utils/gpu.py

```
@@ -233,3 +234,27 @@ def is_hygon_gpu_cmd(pci_addr):
 def reload_hygon_gpu_driver_cmd():
     cmd = "hy-smi --unloaddriver && hy-smi --loaddriver"
     return cmd
+
+def get_vastai_type():
+    r, o, e = bash_roe("lspci | grep Vastai")
```

Ruff (0.8.2)

`bash_roe` may be undefined, or defined from star imports

(F405)